index.html

```diff
@@ -1,4 +1,4 @@
 ---
-layout: index
+layout: index 
```

Comment:
Hello, and thanks for your changes!

Is there a reason this tab space needs to be present here?